### ISSUE TITLE
"Cannot instantiate an arrow function" all of sudden

### ISSUE THREAD

**Issue by michelecocuccio** [codesandbox/codesandbox-client#4300]
# 🐛 bug report

## Description of the problem
I have a demo of my package on CodeSandbox. The package is called "Simple React Lightbox". I never had any issue and the demo was working flawlessly on CodeSandbox. I recently updated my package and everything was working fine. Today I just updated some "plain text" (a caption on an image) and I got an error.

`Cannot instantiate an arrow function`

## How has this issue affected you? What are you trying to accomplish?
I just updated some plain text in my project, I saved and all of sudden I have got this error which comes from "Simple React Lightbox". I never had this error before, I tried to uninstall the dependency or go back to a previous version and I still get the error (previous versions were working as well).

To make sure it had nothing to do with my project, I have tried to test it on a fresh installation of "create react app" and everything works as expected without any error.

I don't know if something has changed with the "CodeSandbox" but the issue wasn't happening before and it doesn't come from my dependency and I am not getting the error elsewhere.

### Link to sandbox: [link]() (optional)
https://codesandbox.io/s/simple-react-lightboxss-39wrb?file=/src/App.js

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      |  https://codesandbox.io/s/simple-react-lightbox-39wrb?file=/src/index.js
| Browser          | Chrome, Safari and FireFox
| Operating System | macOS Catalina 10.15.4 (19E287)

**Comment by CompuIves** [comment]
Hey, I'm sorry for the inconvenience of this. I found the issue, the transpiler has to add a `"use strict";` when it converts an esmodule to commonjs. In your case the function

```
var Rc = (function() { }).bind(void 0)
```

Did not work, because in a strict environment `this` would actually become `void 0`, but if it's not strict `this` will be `Window`. Because of that a check failed.

I've added the fix now in this commit (https://github.com/codesandbox/codesandbox-client/commit/2c9a8f8cd85e41958a54ce49111c873a6e5e2ef9) and it should be live within 20 minutes.
